scripts/launch.sh

```diff
@@ -2,8 +2,8 @@ sudo docker run --name  cctv-service \
     -d \
     -p 8000:8000 \
     -e LANG=C.UTF-8 \
-    -v "/home/publisher/cctv-service/app":/app \
-    -v "/home/publisher/transportation-data-publishing/transportation-data-publishing/data":/data/ \
+    -v "/srv/publisher/cctv-service/app":/app \
+    -v "/srv/publisher/atd-data-publishing/transportation-data-publishing/data":/data/ \
```

Comment:
can't remember if the reason this is changing from `transporation` to `atd` has come up. i'm just curious and not suggesting any change here!

Reply:
I'm not sure -- but if i I had to guess, and it's nothing more than a guess, I suspect this occurred during some organization of the city wide GH account. The governing board is awfully quiet now-days, but it's there, and I figure it was active when it started to deal with organizational issues like this. John or Amenity may know, or it may have been information we lost when Mateo left the team. I have his old seat on the board now.

Reply:
fwiw I dont think the github governing board has ever been active, we didnt have organizational rules, at least none that ODD was aware of. I was added to the group as the ODD dev [when it was starting](https://github.com/cityofaustin/coa-gh-governance/issues/1).

Reply:
oh i see - this change is because the repo name changed at some point. all good and thanks for giving me some background there! 🙏